MastodonSDK/Sources/MastodonUI/View/Content/StatusView+Configuration.swift

```diff
@@ -139,21 +148,27 @@ extension StatusView {
                 return header
             }
 
-            if let replyTo = status.replyTo {
+            if let inReplyToID = status.entity.inReplyToID {
                 // A. replyTo status exist
-                let header = createHeader(name: replyTo.author.displayNameWithFallback, emojis: replyTo.author.emojis.asDictionary)
-                viewModel.header = header
+                
+                /// we need to initially set an empty header, otherwise the layout gets messed up
+                viewModel.header = createHeader(name: "", emojis: [:])
+                /// finally we can load the status information and display the correct header
+                if let authenticationBox = viewModel.authContext?.mastodonAuthenticationBox {
+                    Task { @MainActor in
+                        if let replyTo = try? await Mastodon.API.Statuses.status(
+                            session: .shared,
+                            domain: authenticationBox.domain,
+                            statusID: inReplyToID,
+                            authorization: authenticationBox.userAuthorization
+                        ).singleOutput().value {
+                            let header = createHeader(name: replyTo.account.displayNameWithFallback, emojis: replyTo.account.emojis?.asDictionary ?? [:])
+                            viewModel.header = header
```

Comment:
Could it be that the header-reload-issue I sent you the video from gets triggered by this?

Reply:
When I want to see sensitive content of a reblogged status, I also get this behaviour that the "Reblogged by" disappears

Reply:
We'll open an issue for this weird header-reload.